app/components/ActivityVolonteersList.jsx

```diff
@@ -0,0 +1,112 @@
+var React = require('react')
+var AutoSuggest = require('react-autosuggest');
+var config = require('../../config.json')
+
+
+var AutoSuggestVolonteer = React.createClass ({
+  getInitialState: function () {
+      return {volonteerId: 0};
+  },
+  getSuggestions: function(input, callback) {
+    var query = {
+      suggest: {
+        text: input,
+        completion: {
+          field: "suggest"
+        }
+      }
+    }
+
+    var request = new XMLHttpRequest()
+    request.open('POST', '/getSuggestions', true)
+    request.setRequestHeader('Content-Type', 'application/json')
+    request.onload = function() {
+      if (request.status >= 200 && request.status < 400) {
+        // Success!
+        var resp = request.responseText;
+        console.log(resp)
+        var json = JSON.parse(resp)
+        
+        var suggestions = json.suggest[0].options.map (function (option) {
+          return {
+            id: option.payload.id,
+            name: option.text,
+            email: option.payload.email
+          };
+        })
+
+        setTimeout(callback(null, suggestions), 300);
```

Comment:
Chyba zostało mi to z tego, na czym się wzorowałem na początku. Wydaje mi się, że mogę usunąć ten timeout